db/warehouse/migrate/20251020142047_create_metric_snapshots.rb

```diff
@@ -0,0 +1,105 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+class CreateMetricSnapshots < ActiveRecord::Migration[7.1]
+  def up
+    safety_assured do
+      # Create parent table as partitioned table
```

Comment:
I'm curious to see the size of this table in production. Better to start with partitions and not need the than the other way around.